### PR TITLE
XW-4016 | Update wiki recommendations

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 1]
```diff
@@ -610,60 +640,60 @@ class WikiRecommendations {
...
 			$recommendation['index'] = $index;
 			$index++;
 		}
 
 		return $recommendations;
 	}
 
-	private static function fallbackToSupportedLanguages( $language ) {
-		switch ( $language ) {
+	private static function fallbackToSupportedLanguages($language) {
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: MAJOR Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 2]
```diff
@@ -663,7 +693,7 @@ public static function getRecommendations( $contentLanguage ) {
 	}
 
 	private static function fallbackToSupportedLanguages( $language ) {
```
Comment (WikiaTech) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: MAJOR Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiRecommendations.class.php#L695](https://github.com/Wikia/app/blob/df9005366b63e389b7f8c3922c7f20f7671a2a76/extensions/wikia/Recirculation/WikiRecommendations.class.php#L695): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/Recirculation/WikiRecommendations.class.php [hunk 3]
```diff
@@ -11,565 +11,595 @@ class WikiRecommendations {
...
 		'ja' => [
 			[
 				'thumbnailUrl' => 'https://vignette.wikia.nocookie.net/battlefront/images/3/3b/BF2_IMG.jpg/revision/latest?cb=20170905023120&path-prefix=ja',
 				'title' => 'Battlefront Wiki',
-				'url' => 'http://ja.battlefront.wikia.com',
+				'url' => 'http://ja.battlefront.wikia.com'
 			],
 			[
-				'thumbnailUrl' => 'https://vignette.wikia.nocookie.net/gameofthrones/images/7/75/GOTS7-E5-25.jpg/revision/latest?cb=20170822083703&path-prefix=ja',
+				'thumbnailUrl' => 'http://ja.gameofthrones.wikia.com/wiki/%E7%89%B9%E5%88%A5:%E7%94%BB%E5%83%8F?page=2&file=GOTS7-E5-25.jpg',
```
Comment (latata) [inline extensions/wikia/Recirculation/WikiRecommendations.class.php]: this should be a vignette url

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiRecommendations.class.php#L695](https://github.com/Wikia/app/blob/471ea46b640e540ec388570b1126456d03f388d5/extensions/wikia/Recirculation/WikiRecommendations.class.php#L695): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiRecommendations.class.php#L695](https://github.com/Wikia/app/blob/22dec71cfe541d0996219da88773a7a89a4a6f4f/extensions/wikia/Recirculation/WikiRecommendations.class.php#L695): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by latata — APPROVED** [review]

**Pull request merged**